Makefile

```diff
@@ -236,9 +240,10 @@ endif
 ifeq ($(llvm_version),3.7.1)
 else ifeq ($(llvm_version),3.8.1)
 else ifeq ($(llvm_version),3.9.1)
+else ifeq ($(llvm_version),4.0.0)
 else
   $(warning WARNING: Unsupported LLVM version: $(llvm_version))
-  $(warning Please use LLVM 3.7.1, 3.8.1, or 3.9.1)
+  $(warning Please use LLVM 3.7.1, 3.8.1, 3.9.1, or 4.0.0)
```

Comment:
This wont work with homebrew on the mac until 4.0.1 is in here. 

Based on how we do this, we shouldn't be supporting 4.0.0 we should only support 4.0.1